schema.sql

```diff
@@ -50,3 +50,10 @@ CREATE TABLE visits (
   vet_id INTEGER REFERENCES vets(id),
   visit_date DATE
 );
+
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
+
+ CREATE INDEX owners_idx ON owners (id);
+ CREATE INDEX visits_vet_id_idx ON visits (vet_id);
```

Comment:
Great job team🤝

- [ ] Kindly add all three created indexes to your schema file. You currently have only two indexes in your schema file. Kindly add the third index.